### ISSUE TITLE
rename index.ftl to index.ftlh when upgrading spring boot to 2.2+

### ISSUE THREAD

**Issue by grmkris** [learnk8s/knote-java#3]
Spring Boot recently yet again introduced a breaking change, which leads to the infamous whitelabel error page. They changed the default suffix from .ftl to .ftlh.

https://github.com/spring-projects/spring-boot/issues/15131

So for Spring Boot 2.2.x applications, these extensions have to be updated.

**Comment by salaboy** [comment]
yes.. what the hell.. how do they break the user experience like that.. let me upgrade that now

**Comment by salaboy** [comment]
@grmkris by the way.. did you upgrade the spring boot version.. that's why it failed right?

**Comment by grmkris** [comment]
Yes if I remember correctly

**Comment by salaboy** [comment]
@grmkris ok... if you want to send a PR for that .. I will really appreciate it..

**Comment by grmkris** [comment]
Okay... will do. Just can't promise it will be this week :( Busy life

**Comment by salaboy** [comment]
@grmkris thanks a lot.. same here.. I am doing a webinar today if you want to join :) https://www.linode.com/blog/kubernetes/learnk8s-lab-series/

**Comment by hendisantika** [comment]
I will try to send PR about this Freemarker issue ya.

Please check
